### PR TITLE
Tickets: Added remote tickets

### PR DESCRIPTION
**Pull request by anivorlis**
Added remote tickets on the website.

### CODE REVIEW

File: src/content/pages/tickets.mdx [hunk 1]
```diff
@@ -26,9 +28,13 @@ The conference will follow this broad plan of events:
...
 ticket to give folks maximum flexibility and value for money:
 
-- **Conference Tickets**: Access to Conference & Sprint Weekend (10-14 July).
-  There will be limited access to specific sponsored/special workshops during
-  the Workshop/Tutorial Days (8-9 July).
+- **Conference Tickets** (in-person & remote):
+    - **In-person**: Access to Conference & Sprint Weekend (10-14 July).
+      There will be limited access to specific sponsored/special workshops during
+      the Workshop/Tutorial Days (8-9 July).
+    - **Remote**: remote participation during the Conference Days (10-12 July):
```
Comment (egeakman) [inline src/content/pages/tickets.mdx:35]:
```suggestion
    - **Remote**: Remote participation during the Conference Days (10-12 July):
```

**Comment by egeakman** [comment]
Thanks Aris :heart:

**Comment by egeakman** [comment]
The conflict is gone, we can continue with this one

File: src/content/pages/tickets.mdx [hunk 2]
```diff
@@ -26,9 +28,13 @@ The conference will follow this broad plan of events:
...
+      watch the live talks, keynotes & panels in all 6 tracks, engage in live text-based Q&A,
+      and interact with speakers and in-person attendees in chat channels.
```
Comment (egeakman) [inline src/content/pages/tickets.mdx:37]:
```suggestion
- **Conference Tickets** (in-person & remote):
    - **In-person**: Access to Conference & Sprint Weekend (10-14 July)
      - Limited access to specific sponsored/special workshops
      during the Workshop/Tutorial Days (8-9 July) are also included.
    - **Remote**: Remote participation during the Conference Days (10-12 July)
      - Watch the live talks, keynotes & panels in all 6 tracks,
      - Engage in live text-based Q&A,
      - And interact with speakers and in-person attendees in chat channels.
```

IMHO a structure like this is more readable

**Comment by egeakman** [comment]
Also, we can remove these lines now :tada: 

https://github.com/EuroPython/website/pull/653/files#diff-8eca8cd7b7e03f2f14f1c3955c0921c464f0980727e8e792fd6121b6607c68a0R144-R151

File: src/content/pages/tickets.mdx [hunk 3]
```diff
@@ -13,7 +13,9 @@ subtitle: Ticket Sales for EuroPython 2024 are open!
 
 EuroPython 2024 will be held on 8-14 July. In person, it will be held at
 the [Prague Congress Centre (PCC)](https://www.praguecc.cz/en/homepage), the Czech Republic.
-Details of how to participate remotely will be published soon.
+For those who cannot physically join us but still want to support the community,
+we have the remote ticket option. If you do not have a remote ticket, the livestreaming will
```
Comment (cybit) [inline src/content/pages/tickets.mdx:17]:
```suggestion
we have the remote ticket option. Without a remote ticket, the recordings will
```

**Comment by anivorlis** [comment]
Good, thanks for the comments! 🤗

**Review by egeakman — APPROVED** [review]
LGTM, thanks!

**Pull request merged**